handsontable/src/core.js

```diff
@@ -2636,38 +2636,71 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   /**
-   * Allows altering the table structure by either inserting/removing rows or columns.
-   * This method works with an array data structure only.
+   * The `alter()` method lets you alter the grid's structure
+   * by adding or removing rows and columns at specified positions.
+   *
+   * ::: tip
+   * The `alter()` method works only when your [`data`](@/api/options.md#data)
+   * is an [array of arrays](@/guides/getting-started/binding-to-data.md#array-of-arrays).
+   * :::
+   *
+   * ```js
+   * // above row 10 (by visual index), insert 1 new row
+   * hot.alter('insert_row_above', 10);
+   * ```
+   *
+   *  | Action               | With `index` | Without `index` |
+   *  | -------------------- | ------------ | --------------- |
+   *  | `'insert_row_above'` | Inserts rows above the `index` row. | Inserts rows above the first row. |
+   *  | `'insert_row_below'` | Inserts rows below the `index` row. | Inserts rows below the last row. |
+   *  | `'remove_row'`       | Removes rows, starting from the `index` row. | – |
+   *  | `'insert_col_start'` | Inserts columns before the `index` column. | Inserts columns before the first column. |
+   *  | `'insert_col_end'`   | Inserts columns after the `index` column. | Inserts columns after the last column. |
+   *  | `'remove_col'`       | Removes columns, starting from the `index` column. | – |
```

Comment:
```suggestion
   *  | `'remove_col'`       | Removes columns, starting from the `index` column. | Removes the last column. |
```

Reply:
@warpech – I've already added it (sorry for the delay): [b457b5a](https://github.com/handsontable/handsontable/pull/10046/commits/b457b5a4ddd8bd6bbc3505b745bc7bdc51fcc857)